aks/redis/resources.tf

```diff
@@ -22,6 +22,7 @@ resource "azurerm_redis_cache" "main" {
   minimum_tls_version           = var.azure_minimum_tls_version
   public_network_access_enabled = var.azure_public_network_access_enabled
   redis_version                 = var.server_version
+  zones                         = var.azure_zones
```

Comment:
I get, Inappropriate value for attribute "zones": set of string required.
Also, zones not valid for standard, so only if premium.

Reply:
resolved the data type issues and introduced a condition